compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/CompilerPhaseRunner.java

```diff
@@ -223,6 +231,9 @@ public BLangPackage birGen(BLangPackage pkgNode) {
     private BLangPackage birEmit(BLangPackage pkgNode) {
         return this.birEmitter.emit(pkgNode);
     }
+    private BLangPackage deadCodeAnalyze(BLangPackage pkgNode) {
```

Comment:
```suggestion

    private BLangPackage deadCodeAnalyze(BLangPackage pkgNode) {
```

Reply:
Refactored in commit 18b3939ecaa3fce423297b2a3ad53a9485b75f97